eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -62,14 +62,12 @@ jobs:
             python3 --version
             python3 eng/versioning/set_versions.py --update-type all --build-type client --build-qualifier dev.$(Build.BuildNumber) --artifact-id ${{artifact.name}}
             python3 eng/versioning/set_versions.py --update-type all --build-type data --build-qualifier dev.$(Build.BuildNumber) --artifact-id ${{artifact.name}}
-          condition: eq(variables['SetDevVersion'],'true')
```

Comment:
I why are you removing the SetDevVersion=true condition in the steps here and below? Without the condition this would end up adding the dev version to everything for every build.

Reply:
This is intentional until I merge (I'll add it back in). It just means I can test dev package production in the PR flow (it doesn't publish them).